### ISSUE TITLE
Dependence on Net. Standard

### ISSUE THREAD

**Issue by iguskov1810** [HdrHistogram/HdrHistogram.NET#100]
Hello.
Please tell me, is there a new library planned for release in the near future?
I would like to get rid of Net. Standard in a future version of the library for Net. Core Application.

![image](https://github.com/user-attachments/assets/7b416681-70ea-4753-b8fd-4495669a0738)

**Comment by LeeCampbell** [comment]
Yes. There are some bugs to be resolved, then a release for core will be
made

**Comment by iguskov1810** [comment]
This is cool. We'll wait

Can you tell me approximately when to expect the release?

**Comment by iguskov1810** [comment]
@LeeCampbell Can you answer?

**Comment by LeeCampbell** [comment]
Some work will commence in the next 4 weeks. If the bugs can be resolved,
then a release will be created.
